addons/website_slides_survey/static/src/js/public/components/slide_upload_dialog/slide_upload_category.xml

```diff
@@ -15,15 +15,6 @@
                     togglerClass="'text-dark pe-1'"
                 >
                     <t t-out="displayCertificationValue"/>
-                    <t t-set-slot="bottomArea" t-slot-scope="select">
```

Comment:
Ideally we don't change component structure in stable, but as 17.2 is relatively young and this is highly specific it's acceptable I think.